### ISSUE TITLE
Netdownload not working since the latest Pebble Update on Basalt

### ISSUE THREAD

**Issue by PanicMan** [pebble-examples/pebble-faces#17]
Hello,

netdownload lib is not working on Basalt, because there is now an negative chunk size (-537064848).
Aplite is still working fine... Tested on real and cloudpebble.

Greetings Jack

**Comment by PanicMan** [comment]
Ok, found allready an solution for this, simple change Line 43 fom:
```c++
uint32_t chunk_size = app_message_inbox_size_maximum() - dict_calc_buffer_size(1);
```
to
```c++
uint32_t chunk_size = app_message_inbox_size_maximum() - dict_calc_buffer_size(1, 0);
```
It seems the new firmware has an problem if you omit the buffer size... Works still on aplite.

**Comment by sarfata** [comment]
Fixed by #16
